Sources/BraintreeDropIn/Models/BTDropInResult.m

```diff
@@ -11,10 +11,43 @@
 #import <BraintreeCore/BraintreeCore.h>
 #endif
 
+// Import PayPalDataCollector (Swift) module
+#if __has_include(<Braintree/Braintree-Swift.h>)      // CocoaPods
+#import <Braintree/Braintree-Swift.h>
+
+#elif SWIFT_PACKAGE                                   // SPM
+/* Use @import for SPM support
+ * See https://forums.swift.org/t/using-a-swift-package-in-a-mixed-swift-and-objective-c-project/27348
+ */
+@import PayPalDataCollector;
+
+#elif __has_include("Braintree-Swift.h")              // CocoaPods for ReactNative
+/* Use quoted style when importing Swift headers for ReactNative support
+ * See https://github.com/braintree/braintree_ios/issues/671
+ */
+#import "Braintree-Swift.h"
+
+#else                                                 // Carthage
+#import <PayPalDataCollector/PayPalDataCollector-Swift.h>
+#endif
+
 NSString *const BTDropInResultErrorDomain = @"com.braintreepayments.BTDropInResultErrorDomain";
 
 @implementation BTDropInResult
 
+// For testing
+static Class PayPalDataCollectorClass;
+static NSString *PayPalDataCollectorClassString = @"PPDataCollector";
```

Comment:
I think we can do this with OCMock instead (see [docs](https://ocmock.org/reference/) under the 3.2 Stubs and verification section).

This project doesn't use OCMock, so I tried to import it. OCMock has an SPM bug, which I will report to their GitHub. So rather than use a 2nd package manager for the project to fetch OCMock, I went for this approach.

We can always change this testing situation in the future. Let me know what ya'll think.

Reply:
This approach seems reasonable to me. No need to introduce another package manager for testing.